sda-orch/templates/configmap.yaml

```diff
@@ -0,0 +1,8 @@
+{{- if and .Values.datasetID.external .Values.datasetID.customConfig }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "sda.fullname" . }}-orchestrate-config
+data:
+  {{- ( .Files.Glob ( "files/config.json" )).AsConfig | nindent 2 }}
```

Comment:
I guess this is fine, but some part of me keep expecting the value to be used to find the file with the custom config.

Reply:
I kept them like that for consistency, as we have certificates under `files`, but I can try to put a path under `datasetID.customConfig` if that is any help, or that is what you meant.

Reply:
Not sure what's best, and feel free to ignore this if it seems like to much work, but I think what's tripping me up is that I expect `customConfig` to point to the configuration to use. I don't know if there's any conventions I'm not aware of, but simply turning it into `useCustomConfig` would probably be better to signal yes/no to me. 

(Whereas if it seems desirable to not have it in `config.json`, I guess `customConfigFile` would be clearer.)

Reply:
`useCustomConfig` used. i can try to use `customConfigFile` if the need arises, until i think we can keep it as it is changed

Reply:
👍